sympy/physics/control/lti.py

```diff
@@ -3838,6 +3838,107 @@ def num_outputs(self):
         """
         return self._D.rows
 
+    def state_vector(self, initial_conditions=None, input_vector=None, var=None):
+        r"""
+        Returns the State vector `x` given by the solution of the state equation
+            x'(t) = A * x(t) + B * u(t)
+
+        Parameters
+        ============
+
+        initial_conditions : Matrix
+            The initial conditions of `x` state vector.
+        input_vector : Matrix
+            The input vector for state space.
+        var : Symbol
+            The symbol representing time.
+
+        Examples
+        ==========
+
+        >>> from sympy import Matrix, Symbol
+        >>> from sympy.physics.control import StateSpace
+        >>> A = Matrix([[-2, 0], [1, -1]])
+        >>> i = Matrix([2, 3])
+        >>> t = Symbol('t')
+        >>> ss = StateSpace(A)
+        >>> ss.state_vector(initial_conditions=i, var=t)
+        Matrix([
+        [            2*exp(-2*t)],
+        [5*exp(-t) - 2*exp(-2*t)]])
+
+        References
+        ==========
+        .. [1] https://web.mit.edu/2.14/www/Handouts/StateSpaceResponse.pdf
+        .. [2] https://docs.sympy.org/latest/modules/solvers/ode.html#sympy.solvers.ode.systems.linodesolve
+        """
+
+        from sympy.solvers.ode.systems import linodesolve
+        from sympy import Eq, solve
```

Comment:
It may be better to move the imports to the top of the module, because it is not likely that sympy core or solvers may reference `physics`